README.rst

```diff
@@ -87,10 +87,10 @@ The following configuration parameters are available:
 Key                        Default         Description
 =========================  ==============  =========================================================================================================================
 actingPartyCn                              The acting party Common name used in contracts
-address                    localhost:1323  Interface and port for http server to bind to
+address                    localhost:1323  Interface and port for http server to bind to, default: localhost:1323
```

Comment:
Why listing the default in the description? Seems redundant information since it is already in the Default column.

Reply:
> Why listing the default in the description? Seems redundant information since it is already in the Default column.

The "Default" column is not shown when the command usage (CLI) is printed, so then you need the default value in the description.